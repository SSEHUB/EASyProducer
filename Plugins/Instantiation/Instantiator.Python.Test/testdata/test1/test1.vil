vilScript test1 (Project source, Configuration config, Project target) {

    version v0;

    main(Project source, Configuration config, Project target) = : {   	
    	vilTemplateProcessor("$target/test1.vtl", config, "$target/test1.py");
    }
}