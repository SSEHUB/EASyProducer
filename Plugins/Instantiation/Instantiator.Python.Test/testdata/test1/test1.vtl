@indent(indentation = 4, additional = 0)
@format(profile="python", lineLength = 120)
template test1(Configuration config, PythonCodeArtifact target) {
    def main(Configuration config, PythonCodeArtifact target) {
    	target.comment("test comment");
    	target.addAssign("a", "42069");
    	target.addAssign("b", "str", '"some string"');
    }
}
