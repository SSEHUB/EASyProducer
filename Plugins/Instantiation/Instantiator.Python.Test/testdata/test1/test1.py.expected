"""test comment"""

a = 42069
b: str = "some string"
